internal/testcatalogs/demo_catalog.json

```diff
@@ -0,0 +1,365 @@
+[
```

Comment:
Why duplicate?

Reply:
I thought we should have one catalog just for the tests in "internal/testcatalogs" and the other one in "catalogs" so that we have one at the start of the Clouditor. 

You would remove the one in "catalogs"?!?

Reply:
I would remove the one in testcatalogs